buildscripts/import-control.xml

```diff
@@ -148,6 +148,7 @@ General guidelines on imports:
     <subpackage name="stats">
       <allow pkg="io.opencensus.stats"/>
```

Comment:
Are `stats` and `tags` still needed?

Reply:
Both are still needed for other two exporters (prometheus and signalFx), I will remove that once we update these exporters to use Metrics API.